constraints.pro

```diff
@@ -65,15 +65,15 @@ gen_enforced_field(WorkspaceCwd, 'repository.url', 'https://github.com/MetaMask/
 % The license for the package must be specified.
 gen_enforced_field(WorkspaceCwd, 'license').
 
-% The entrypoint for the package must be `./dist/index.js`.
-gen_enforced_field(WorkspaceCwd, 'main', './dist/index.js').
+% The entrypoint for the package must be `./dist/cjs/index.js`.
+gen_enforced_field(WorkspaceCwd, 'main', './dist/cjs/index.js').
```

Comment:
Should we add more constraints to check:

- `module`
- `exports`

Reply:
(I'm also noticing that we don't have constraints to check `scripts` like we do in `core`, but that's out of scope)